election/packages/ge2020/exitpolls/index.js

```diff
@@ -1,10 +1,12 @@
-const fs = require('fs');
+const fs = require('fs')
 
-const contents = fs.readFileSync(__dirname + '/exitpolls.json', 'UTF-8');
-const body = JSON.parse(contents);
+let contents
 
 function main(args) {
-    return { body };
+	if (!contents) {
+		contents = fs.readFileSync(__dirname + '/exitpolls.json', 'UTF-8')
```

Comment:
why move these back into the function? If you lift them to be constants then the file is read once and you don't need the `!contents` check.

Reply:
Earlier it was that way only, I had to change it while writing unit tests. 
While testing the main method, fs.readFileSync needs to be mocked, but mocking is ineffective unless, it is contained in the function being tested or being passes as a dependency. 
I will move it out and pass as a param to main or export 'content' as well so that it could be mocked separately. Thanks!